pontoon/base/views.py

```diff
@@ -104,6 +105,17 @@ def locale(request, locale, template='locale.html'):
     return render(request, template, data)
 
 
+def locales(request, template='locales.html'):
+    """Localization teams."""
+    log.debug("Localization teams.")
+
+    data = {
```

Comment:
Nitpicking: Might as well pass this directly to render instead of bothering with the variable.